ckan/controllers/group.py

```diff
@@ -227,15 +240,15 @@ def _read(self, id, limit):
         def search_url(params):
             if group_type == 'organization':
```

Comment:
I think group['is_organization'] and group_type=='organization' are interchangeable. I don't see a situation where you'd have one and not the other - or can you?